docs/pages/versions/unversioned/sdk/video.mdx

```diff
@@ -139,6 +139,100 @@ const styles = StyleSheet.create({
 
 </SnackInline>
 
+## Preloading videos
+
+While another video is playing, a video can be loaded before being shown in the view. This allows for quicker transitions between subsequent videos and better user experience.
+
+In order to preload a video, a `VideoPlayer` with a video source has to be created. Even when the player is not connected
+to a `VideoView`, it will fill the buffers, and once it is connected to a `VideoView` it will be able to start playing without buffering.
```

Comment:
```suggestion
To preload a video, you have to create a`VideoPlayer` with a video source. Even when the player is not connected to a `VideoView`, it will fill the buffers. Once it is connected to the `VideoView`, it will be able to start playing without buffering.
```

- Let's use active voice to improve readability in the first sentence.
- Let's split the second sentence.